backend/venv/Lib/site-packages/click/testing.py

```
@@ -0,0 +1,565 @@
+        os.chdir(dt)
+
+        try:
+            yield dt
+        finally:
+            os.chdir(cwd)
+
+            if temp_dir is None:
+                try:
+                    shutil.rmtree(dt)
+                except OSError:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(OSError)`

(SIM105)